samplesheets/views.py

```diff
@@ -885,6 +909,190 @@ def handle_alerts_deactivate(cls, irods_request, app_alerts=None):
                 )
             )
 
+    @classmethod
+    def accept_request(
+        cls,
+        irods_request,
+        project,
+        request,
+        timeline=None,
+        taskflow=None,
+        app_alerts=None,
+    ):
+        """
+        Accept an iRODS accept request.
+
+        :param irods_request: IrodsDataRequest object
+        :param project: Project object
+        :param request: Request object
+        :param timeline: TimelineAPI instance or None
+        :param taskflow: TaskflowAPI instance or None
+        :param app_alerts: AppalertsAPI instance or None
+        :raise: FlowSubmitException if taskflow submission fails
+        """
+        tl_event = None
+        description = 'accept iRODS data request {irods_request}'
+        status = 'OK'
```

Comment:
These two lines should go under `else` after the `ACCEPTED` check, or the other way around.